frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -167,7 +170,20 @@ export const rerunPipelineAndStay: KebabAction = (kind: K8sKind, pipelineRun: Pi
 };
 
 export const rerunPipelineAndRedirect: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
-  return rerunPipeline(kind, pipelineRun, null, { onComplete: handlePipelineRunSubmit });
+  return rerunPipeline(kind, pipelineRun, null, {
+    onComplete: handlePipelineRunSubmit,
+    label: 'Start Last Run',
+  });
+};
+
+export const rerunPipelineFromDetailPage: KebabAction = (
```

Comment:
FWIW (thought experiment, since this function is going away), let's avoid explaining where something is used. We should focus on what it is going to do so we can reuse it later.

Reply:
Please rename this
```suggestion
export const rerunPipelineAndRedirect: KebabAction = (
```
that'll satisfy my above comment.

Reply:
```rerunPipelineAndRedirect``` function is already there.

Reply:
Done.